packages/components/scripts/jobs/createChecksums.js

```diff
@@ -0,0 +1,64 @@
+import { execSync } from 'child_process';
+import fs from 'fs';
+import path from 'path';
+
+
+import { job } from './shared.js';
+
+/**
+ * This function will create the events/events.ts file out of all vendored events
```

Comment:
Could you maybe adjust the comment, its misleading

Reply:
It's not misleading, it's wrong. 😆 Copy pasta, mmmmmmmmh. 🍝